app/mailers/host_mailer.rb

```diff
@@ -14,7 +14,7 @@ def summary(options = {})
 
     total_metrics = load_metrics(host_data)
     total = 0
-    total_metrics.values.each { |v| total += v }
+    total_metrics.each_value { |v| total += v }
```

Comment:
Ruby 2.4 introduced `Enumerable#sum` which is nicer:
```suggestion
total = total_metrics.each_value.sum
```